internal/factsengine/gatherers/corosynccmapctl_test.go

```diff
@@ -136,55 +136,43 @@ func (suite *CorosyncCmapctlTestSuite) TestCorosyncCmapctlGatherer() {
 
 	factRequests := []entities.FactRequest{
 		{
-			Name:     "quorum_provider",
+			Name:     "nodes",
 			Gatherer: "corosync-cmapctl",
-			Argument: "quorum.provider",
+			Argument: "nodelist.node",
```

Comment:
What happens if you add a new request with `nodelist.node.0` as argument?
I think it might fail as you store the "list" entries as maps

Reply:
Requesting `nodelist.node.0` returns a map containing two keys
`node_id` and `ring_addr` 

```golang
...
	Name: "nodes",
	Value: &entities.FactValueMap{
		Value: map[string]entities.FactValue{
			"nodeid":     &entities.FactValueInt{Value: 1},
			"ring0_addr": &entities.FactValueString{Value: "10.80.1.11"},
		},
	},
...
```